modules/ort/pom.xml

```diff
@@ -42,6 +56,32 @@
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
         </dependency>
+        <!-- ################################ flattened ORT dependencies ################################ -->
+        <dependency>
+            <groupId>com.github.heremaps.oss-review-toolkit</groupId>
+            <artifactId>analyzer</artifactId>
+            <version>jitpack-e972a4f3bb-1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.heremaps.oss-review-toolkit</groupId>
+            <artifactId>model</artifactId>
+            <version>jitpack-e972a4f3bb-1</version>
```

Comment:
We try to define versions in the `dependencyManagement` in the top level pom. And if a version appears multiple times we extract it into a variable.

Reply:
Done.